torchvision/prototype/transforms/kernels/_augment.py

```diff
@@ -0,0 +1,52 @@
+from typing import Tuple
+
+import torch
+from torchvision.transforms import functional as _F
+
+
+erase_image = _F.erase
+
+
+def _mixup(input: torch.Tensor, batch_dim: int, lam: float, inplace: bool) -> torch.Tensor:
+    if not inplace:
+        input = input.clone()
+
+    input_rolled = input.roll(1, batch_dim)
+    return input.mul_(lam).add_(input_rolled.mul_(1 - lam))
+
+
+def mixup_image(image_batch: torch.Tensor, *, lam: float, inplace: bool = False) -> torch.Tensor:
```

Comment:
I'm a bit confused over the use of default values. What happened to them and why this method is allowed to have one?